### ISSUE TITLE
Add prey to f-droid store

### ISSUE THREAD

**Issue by wuniversales** [prey/prey-android-client#152]

**Comment by tomkel** [comment]
yes please!

**Comment by fabianski7** [comment]
https://github.com/prey/prey-android-client/issues/84

**Comment by IzzySoft** [comment]
Essentially #84 indeed is holding this back: the app uses Firebase, FCM and Google Services (eg. for location), so for F-Droid to pick it up it would need a build flavor that comes without those (but then could add the camouflage back – it's not the Android OS not permitting such a feature, but rather Google Play AFAIK).

For starters, the APK could be attached to [`releases/`](https://github.com/prey/prey-android-client/releases) here – so it would at least be reachable for those outside of Google's walled garden (like owners of Huawei devices, or users of Google-free custom ROMs). I might even add it to [my repo](https://apt.izzysoft.de/fdroid) then, though I normally do not add security apps with Firebase bindings (with proprietary stuff you can never be sure what it does) – and especially not with proprietary analytics libraries (here: Crashlytics).

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
Please consider this, not least because although much software is available via not merely the main, but 3rd-party repositories, and the userbase consists of many users, you would possess monopoly, and allow superior control of what your software is dependent upon.

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
However, @prey, please be aware that although @IzzySoft is correct that your repository must be modified for it to be accepted into F-Droid's default repository, they were merely referring to its default repository. You probably possess experience with Debian and Fedora: how many people do not need to utilise Fedora's COPR ever? Few. Less never add Personal Package Archives, both of which are ultimately identical to their native "repositories", and Fedora's RPMFusion and Ubuntu's proprietary repositories are identically as frequently utilised. Consequently, I believe that you should at least expose the binary .APK installers that your CI currently creates via an F-Droid repository, because it would allow me to invest in your software, which I am alternatively unable to do because of my lack of Google Play Store and rational dissuasion to manual updating of software.

**Comment by IzzySoft** [comment]
That's correct, an own repo is a fully valid approach. As for my repo, I'd just need the APK attached to its corresponding release; inclusion criteria for my repo are a little more relaxed than those on F-Droid – so while I recommend to leave all proprietary stuff out (else it's not really FOSS), I usually close an eye on things like FCM (unless the app covers sensitive material, such as apps for health, finances, children etc).

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
@ghost, has "http://github.com/prey/prey-android-client/issues/82#issuecomment-93989938" progressed?
